include/modem/lte_lc.h

```diff
@@ -209,40 +209,41 @@ enum lte_lc_evt_type {
 	/**
 	 * Network registration status.
 	 *
-	 * The associated payload is the @c nw_reg_status member of type @ref lte_lc_nw_reg_status
-	 * in the event.
+	 * The associated payload is the @c lte_lc_evt.nw_reg_status member of type
+	 * @ref lte_lc_nw_reg_status in the event.
 	 */
 	LTE_LC_EVT_NW_REG_STATUS,
 
 	/**
 	 * PSM parameters provided by the network.
 	 *
-	 * The associated payload is the @c psm_cfg member of type @ref lte_lc_psm_cfg in the event.
+	 * The associated payload is the @c lte_lc_evt.psm_cfg member of type
+	 * @ref lte_lc_psm_cfg in the event.
 	 */
 	LTE_LC_EVT_PSM_UPDATE,
 
 	/**
 	 * eDRX parameters provided by the network.
 	 *
-	 * The associated payload is the @c edrx_cfg member of type @ref lte_lc_edrx_cfg in the
-	 * event.
+	 * The associated payload is the @c lte_lc_evt.edrx_cfg member of type
+	 * @ref lte_lc_edrx_cfg in the event.
 	 */
 	LTE_LC_EVT_EDRX_UPDATE,
 
 	/**
 	 * RRC connection state.
 	 *
-	 * The associated payload is the @c rrc_mode member of type @ref lte_lc_rrc_mode in the
-	 * event.
+	 * The associated payload is the @c lte_lc_evt.rrc_mode member of type
+	 * @ref lte_lc_rrc_mode in the event.
 	 */
 	LTE_LC_EVT_RRC_UPDATE,
 
 	/**
 	 * Current cell.
 	 *
-	 * The associated payload is the @c cell member of type @ref lte_lc_cell in the event.
-	 * Only the @c cell.tac and @c cell.id members are populated in the event payload. The rest
-	 * are expected to be zero.
+	 * The associated payload is the @c lte_lc_evt.cell member of type
+	 * @ref lte_lc_cell in the event. Only the @c lte_lc_cell.tac and @c lte_lc_cell.id
```

Comment:
With latest updates, I've just added proper references for tac, cell ID and some time fields further below. Other than that the PR hasn't been touched. Now I'm waiting for the CI and doc build to confirm that references look good in generated doc.